fb.txt

```diff
@@ -8,6 +8,7 @@
 facebook.com##div[data-pagelet^=FeedUnit_]:has(a[aria-label=Sponsored])
 facebook.com##div[data-pagelet^=FeedUnit_]:has(a[aria-label=label]:has-text(S):has-text(p):has-text(o))
 facebook.com##div[data-pagelet="RightRail"] > div > span > div:has-text(Sponsored)
+facebook.com##div[role=article]:has(a[aria-label=Sponsored])
```

Comment:
```suggestion
facebook.com##div[role=article]:has(a[aria-label=Sponsored])
facebook.com##div[role=article]:has(a[aria-label=label]:has-text(S):has-text(p):has-text(o))
```
aria-label is only equal to "Sponsored" after you right click on it. otherwise its equal to "label" and you need to check for the fucky split up text in it.